AlphaWallet/Localization/es.lproj/Localizable.strings

```diff
@@ -322,3 +322,5 @@
 "dappBrowser.myDapps.edit.title.label" = "Title";
 "dappBrowser.myDapps.edit.url.label" = "Address";
 "dappBrowser.clearMyDapps" = "Remove Dapp?";
+
+"blockchain.XDAI" = "blockchain.XDAI";
```

Comment:
Shall we just use the English value, so it'll look nicer if the user sees it? Or does it have to be like that for the localisation service to work with it?

i.e.

```
"blockchain.XDAI" = "xDAI Chain";
```

Reply:
It is using the english version, you are correct about the localisation service, appcode added that in for me once I added it into the English file